service/src/main/kotlin/xyz/lebkuchenfm/domain/commands/processors/SayCommandProcessor.kt

```diff
@@ -8,13 +8,13 @@ import xyz.lebkuchenfm.domain.commands.model.Command
 import xyz.lebkuchenfm.domain.commands.model.CommandProcessingResult
 import xyz.lebkuchenfm.domain.eventstream.Event
 import xyz.lebkuchenfm.domain.eventstream.EventStream
-import xyz.lebkuchenfm.domain.radiopersonality.speechsynthesis.TextToSpeechProvider
```

Comment:
I wish there was `package private` visibility modifier to now hide `speechsynthesis` from the rest of the code